tweb/src/main/java/io/fundreqest/platform/tweb/infrastructure/mav/Initializer.java

```diff
@@ -10,32 +11,39 @@
 @Component
 public class Initializer {
 
+    private final boolean initializeCacheOnStartup;
+
+    public Initializer(@Value("${io.fundrequest.cache.initialize-on-startup:true}") final boolean initializeCacheOnStartup) {
```

Comment:
should be default false and property added to the servers

Reply:
- [x] Done in code
- [x]  Done on servers